code-challenges.test.js

```diff
@@ -16,35 +16,142 @@
 // a) Create a test with an expect statement using the variable provided. 
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
-
+describe("arrShuffle", () => {
+    it("takes in an array, removes the first item from the array and shuffles the remaining content.", () => {
+        const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+        const outPut1 = ["yellow", "blue", "pink", "green"]
+            // can be different order
+        const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+        const outPut2 = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+            // can be different order
+
+      expect(arrShuffle(colors1)).toEqual(expect.arrayContaining(outPut1))
+      expect(arrShuffle(colors2)).toEqual(expect.arrayContaining(outPut2))
+    })
+  })
+// test 1 -    ReferenceError: arrShuffle is not defined
+// test 2 -    Pass
+  
 
 // b) Create the function that makes the test pass.
+  //create function called arrShuffle
+  //Pass arrShuffle an argument of arr
+  //within function, use .shift mutator to remove initial element in array
+  //return shuffled array using .sort built-in method passed an anonymous function of Math.random subtracted by 0.5. This will generate either a positive or negative number for .sort to use each comparison, somewhat randomizing output.
+    //NOTE: This problem stumped me for a few hours. Several attempts at .maps and for loops later I couldn't invent a satisfying way to randomize the array. This answer was found through outside resources and put into my function. There is apparently a more efficient and truly random code that can be used, but I didn't understand the logic well enough to feel good about using it as an answer here.
```

Comment:
Excellent work on this problem!  It is a tricky one, especially with the test, but you crushed it!

Reply:
Thank you